go-controller/pkg/node/vrfmanager/vrf_manager_test.go

```diff
@@ -116,6 +116,9 @@ var _ = ginkgo.Describe("VRF manager", func() {
 			enslaveLinkMock1.On("Type").Return("dummy")
 			err = c.reconcile()
 			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+			// Invoke reconcile again to ensure both vrf links in sync.
+			err = c.reconcile()
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
```

Comment:
hmm I see you are testing error didn't happen... but shouldn't we check the vrf was not removed? I don't see the ensure vrf links in sync is happening...? what am I missing..

Reply:
spoke with Peri, he will add another test to check reconcile+ vrf deletion's better

Reply:
yes @tssurya , added a test now. PTAL.